src/content/plugins/split-chunks-plugin.md

```diff
@@ -303,7 +303,7 @@ module.exports = {
 
 #### `splitChunks.cacheGroups.{cacheGroup}.filename`
 
-`string`
+`string` `function (chunkData): string`
```

Comment:
Adding chunkData's type too would be nice if it can be done

Reply:
@EugeneHlushko It this possible?

Reply:
Don't have it, looks like output.filename doesn't have it as well. just a small example. Someone can investigate that one later, would have to set up a project and use --inspect